_posts/2018/2018-04-01-stop-robots-crawlers-triggering-errors-rails.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Stop robots and crawlers causing errors in your Rails application'
+description: 'Beware the remorseless robot army.'
+layout: article
+category: ruby
+image:
+  base: '2018/stop-robots-crawlers-triggering-errors-rails'
+  alt: 'BB-8'
+  source: 'https://unsplash.com/photos/C8VWyZhcIIU'
+  credit: 'Joseph Chan'
+---
+
+When a Rails application can't find a record, it throws a 404 error. This is a standard HTTP code for browsers (and other clients) meaning 'not found'.
```

Comment:
Get rid of the parentheses and the stuff in it...extra info that's not necessary I think...